src/components/tasks/card/index.tsx

```diff
@@ -56,9 +61,9 @@ const Card: FC<Props> = ({
   const iconHeight = '25px';
   const iconWidth = '25px';
 
-  const date:string = !!localEndsOn ? getDateInString(localEndsOn) : '';
+  const date: string = !!localEndsOn ? getDateInString(localEndsOn) : '';
```

Comment:
Lint fix might have caused these let's try to keep the delta minimum.

Reply:
what is delta minimum?

Reply:
delta literally means changes, so there should be no changes in non required lines. i.e all changes should be related to required stuff.

Reply:
ok ok

Reply:
fixed linting issues